baby_bingo.py

```diff
@@ -0,0 +1,60 @@
+
+########## SUPPORTING PACKAGES ##########
+import random
+import re
+import sys
+
+
+########## SUPPORTING FUNCTIONS ##########
+
+def remove_number_prefix(my_string):
+    m = re.match('^[0-9]*[.]\s(.*)', my_string)
+    return(m.group(1))

```

Comment:
there is no check to see whether this regex has successfully matched anything.
it's possible that `m.group(1)` doesn't exist, in which case the script will crash.